compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/expressions/EqualsExpressionNode.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Why license year is mentioned as 2017?

Reply:
`EqualsExpressionNode.java` class is no longer available.